### ISSUE TITLE
Suggestion: raise an error on attempt to access non-existant (nil) variables

### ISSUE THREAD

**Issue by Sparker95** [dedmen/ArmaDebugEngine#2]
Arma doesn't raise an error if we call a code-variable which is not defined:
`0 call fnc_codeDoesNotExist; // No error`
Which causes obvious inconvenience.

More thoughts about this feature:

- Does it need to work with private variables too?
- Will it work with `with uiNamespace do` ?

**Comment by dedmen** [comment]
> Will it work with with uiNamespace do ?

yes, any global variable no matter what namespace is currently active.

> Does it need to work with private variables too?

That needs to be configurable.. somehow. ADE doesn't have a config thing yet.
So maybe it will just be enabled/disabled via script commands.
